Cargo.toml

```diff
@@ -181,3 +181,7 @@ required-features = ["rm0433"]
 [[example]]
 name = "spi-dma-rtic"
 required-features = ["rm0433","rt"]
```

Comment:
Since CRC isn't standartized via embedded_hal yet it would be nice if we could feature gate it so only people who actually want to use it have to compile the code and the docs don't get too cluttered.

Reply:
Added feature gate.